### ISSUE TITLE
${''} in new line will invalidate the whole css

### ISSUE THREAD

**Issue by ron-liu** [zeit/styled-jsx#235]
See the following example:

```jsx
import css from './pp.js'
export default () => <div className="pp">
	<style jsx>{css}</style>
</div>)
```
pp.js 
```jsx
export default `
	.pp {
		background-color: red;
		width: 100px;
		height: 100px;
	}
        ${''} /* no matter what is put, even nothing like this one, it will cause the whole style-jsx invalid*/
`
```

The stranger thing is the following is valid
```jsx
export default `
	.pp {
		background-color: red;
		width: 100px;
		height: 100px;

		border: blue ${'13px'} solid  /* this one works*/
	}
`

```

By the way, I am using next.js 3 beta11 with syled-jsx 1.0.4.

**Comment by giuseppeg** [comment]
I am fixing this in #227

**Comment by giuseppeg** [comment]
fixed in #227
